doc/uwa-spec.md

```diff
@@ -156,6 +156,16 @@ A Verifier can validate web attestations URI `uwa://[JWS]` attached to a web sit
 1. Checking that the token expiration (encoded as a number of days after the Unix epoch) is after the presentation message's timestamp (encoded in milliseconds).
 1. Extracting the token label, consisting of a string `<type>: <value>` where the `<type>` is extracted from the Issuer parameters specification's `lblType` value, and the `<value>` is extracted by looking up the specification's `lblValues[lbl]` using the token's `lbl` value.
 
+## QR encoding
+
+A web attestations URI `uwa://[JWS]` can be encoded into a QR code by following these steps:
+1. Split the 3-part `[JWS]` string in its period-separated parts: `[JWS_HEADER]`, `[JWS_PAYLOAD]`, and `[JWS_SIGNATURE]`.
+1. Decode the base64url-encoded JWS parts into their corresponding byte representation: `[JWS_HEADER_BYTES]`, `[JWS_PAYLOAD_BYTES]`, and `[JWS_SIGNATURE_BYTES]`.
+1. Create a QR code with medium error correction (level M) with 4 byte-encoded segments (chunks):  `uwa://`, `[JWS_HEADER_BYTES]`, `[JWS_PAYLOAD_BYTES]`, and  `[JWS_SIGNATURE_BYTES]`.
```

Comment:
try 
_with four, byte-encoded, segments_
instead of
_with 4 byte-encoded segments_

I initially read it as 4-byte-encoded and was confused

Reply:
Updated